Revise note re degenerate xform and rad grd (#112)
googlefonts/colr-gradients-spec: colr-gradients-spec.md

*__Figure 5:__ Radial gradients with one circle contained within the other. (Illustration generated from [images/radial_gradients_circle_within_circle.svg](images/radial_gradients_circle_within_circle.svg).)*

*__Note:__ When a radial gradient is nested below a transformation which flattens

> the circles so that they resemble lines, provided that line is not colinear with

Reviewer: ```suggestion
the circles so that they resemble lines, provided that the lines are not colinear with
```
